README.md

```diff
@@ -72,329 +93,235 @@ Mon Aug  5 17:52:22 2024
+
+1. Optionally, build and push custom, up-to-date controller and daemonset images from source:
 
-- Delete mig slices using the command
-
-```sh
-sudo nvidia-smi mig -dci && sudo nvidia-smi mig -dgi
-
-Successfully destroyed compute instance ID  0 from GPU  0 GPU instance ID  9
-Successfully destroyed compute instance ID  0 from GPU  0 GPU instance ID 10
-Successfully destroyed compute instance ID  0 from GPU  0 GPU instance ID  3
-Successfully destroyed compute instance ID  0 from GPU  0 GPU instance ID  2
-Successfully destroyed compute instance ID  0 from GPU  1 GPU instance ID  9
-Successfully destroyed compute instance ID  0 from GPU  1 GPU instance ID 10
-Successfully destroyed compute instance ID  0 from GPU  1 GPU instance ID  3
-Successfully destroyed compute instance ID  0 from GPU  1 GPU instance ID  2
-Successfully destroyed GPU instance ID  9 from GPU  0
-Successfully destroyed GPU instance ID 10 from GPU  0
-Successfully destroyed GPU instance ID  3 from GPU  0
-Successfully destroyed GPU instance ID  2 from GPU  0
-Successfully destroyed GPU instance ID  9 from GPU  1
-Successfully destroyed GPU instance ID 10 from GPU  1
-Successfully destroyed GPU instance ID  3 from GPU  1
-Successfully destroyed GPU instance ID  2 from GPU  1
+```console
+# IMG=<registry>/<controller-image>:<tag> IMG_DMST=<registry>/<daemonset-image>:<tag> make docker-build docker-push
 ```
 
-- Create placeholder slice to make k8s-device-plugin happy using the command
+Example:
+```
+# IMG=quay.io/example/instaslice2-controller:l.0 IMG_DMST=quay.io/example/instaslice2-daemonset:1.0 make docker-build docker-push
+```
 
-```sh
-sudo nvidia-smi mig -cgi 3g.20gb -C
-Successfully created GPU instance ID  2 on GPU  0 using profile MIG 3g.20gb (ID  9)
-Successfully created compute instance ID  0 on GPU  0 GPU instance ID  2 using profile MIG 3g.20gb (ID  2)
-Successfully created GPU instance ID  2 on GPU  1 using profile MIG 3g.20gb (ID  9)
-Successfully created compute instance ID  0 on GPU  1 GPU instance ID  2 using profile MIG 3g.20gb (ID  2)
+**Note**: You can use Podman instead of Docker to build images, just set `CONTAINER_TOOL=podman` before the image-related make targets.
+
+Cross-platform or multi-arch images can be built and pushed using `make docker-buildx`. When using Docker as your container tool, make
+sure to create a builder instance. Refer to [Multi-platform images](https://docs.docker.com/build/building/multi-platform/)
+for documentation on building mutli-platform images with Docker. You can change the destination platform(s) by setting `PLATFORMS`, e.g.:
+
+```console
+# PLATFORMS=linux/arm64,linux/amd64 make docker-buildx
 ```
 
-- Run the below command to patch device plugin with configmap created by the setup script. For OpenShift replace clusterpolicies.nvidia.com/cluster-policy to clusterpolicies.nvidia.com/gpu-cluster-policy and namespace to nvidia-gpu-operator
+2. Install the CRDs into the cluster
+
+**TODO**: Is this still required?
```

Comment:
@asm582 is it still required? The demo seems to run successfully without explicitly installing the CRDs.

Reply:
This is not required explicitly, make deploy will work and deploy CRD plus the controllers. I use the below command to build the dev images, push to a custom registry and later deploy images on KinD cluster:  
```
make docker-build && make docker-push && make deploy
```

Reply:
@asm582 There is a way to load local images into a Kind cluster without pushing them into a registry. This allows a faster turnover, and eliminates the need in a remote registry for those who want to experiment. Do you think we should implement this? Or are there plans to move away from Kind?

Reply:
I would create a GitHub issue for it and visit it later